src/accelerate/utils/operations.py

```diff
@@ -351,6 +344,12 @@ def wrapper(*args, **kwargs):
             tensor = kwargs["tensor"]
         else:
             tensor = args[0]
+        state = PartialState()
```

Comment:
No, we already instantiate the state here: https://github.com/huggingface/accelerate/blob/main/src/accelerate/utils/operations.py#L287C27-L287C27

Reply:
(And probably why the tests are failing)

Reply:
Yes makes sense, I modified it to use the existing PartialState() instead. Thanks